src/tsconfig-base.json

```diff
@@ -12,14 +12,16 @@
         "composite": true,
         "noEmitOnError": true,
 
-        "strictNullChecks": true,
-        "noImplicitAny": true,
-        "noImplicitThis": true,
-        "strictPropertyInitialization": true,
+        "strict": true,
         "noUnusedLocals": true,
         "noUnusedParameters": true,
+        "allowUnusedLabels": false,
+
+        "useUnknownInCatchVariables": false,
+        "strictBindCallApply": false,

```

Comment:
Maybe group these with `strict`, then group the "stylistic" checks together.

Reply:
Done.